### PR TITLE
Initialize supplementing files

### PR DESCRIPTION
**Pull request by stellargo**
Adds

- .gitattributes
- .github/CODEOWNERS
- .github/release-drafter.yml
- .mvn/maven.config
- Jenkinsfile

Updates 

- pom.xml

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -35,13 +35,15 @@ THE SOFTWARE.
     <groupId>io.jenkins.plugins</groupId>
```
Comment (oleg-nenashev) [inline pom.xml:35]: Parent POM could be updated to 4.4

File: pom.xml [hunk 2]
```diff
@@ -35,13 +35,15 @@ THE SOFTWARE.
...
     <artifactId>postgresql-fingerprint-storage</artifactId>
     <version>1.0-SNAPSHOT</version>
-    <packaging>pom</packaging>
+    <packaging>hpi</packaging>
 
     <properties>
         <jenkins.version>2.245</jenkins.version>
```
Comment (oleg-nenashev) [inline pom.xml:41]:
```suggestion
        <jenkins.version>2.248</jenkins.version>
```

You will need new APIs, right?

File: .github/release-drafter.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+version-template: '$MAJOR.$MINOR-alpha-$PATCH'
+name-template: 'v$RESOLVED_VERSION 🌈'
+tag-template: 'v$RESOLVED_VERSION'
+
+categories:
```
Comment (oleg-nenashev) [inline .github/release-drafter.yml:5]: Why don't you use the standard Release Drafter configuration?

File: pom.xml (hunk 2 above)
Comment (stellargo) [reply pom.xml:41]: I am getting some troubles with 2.248. I am looking into resolving them.

Comment (stellargo) [reply pom.xml:41]: Resolved. Maven had downloaded an empty 2.248 jar, which was causing this issue.

File: .github/release-drafter.yml (hunk 3 above)
Comment (stellargo) [reply .github/release-drafter.yml:5]: Got it from https://github.com/release-drafter/release-drafter#example. I personally think it is more informative, and uses labels to automatically put the change in the correct category. 
I am not aware of the standard one, but I don't have any strong preferences :)

Comment (stellargo) [reply .github/release-drafter.yml:5]: @oleg-nenashev Would you recommend using this https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/.github/release-drafter.yml ?

File: pom.xml (hunk 1 above)
Comment (stellargo) [reply pom.xml:35]: So I looked into it, seems it wasn't `jenkins.version` whose support was dropped, but rather `jenkins-core.version`. Ill allow dependabot to go ahead and do this.

File: .github/release-drafter.yml (hunk 3 above)
Comment (oleg-nenashev) [reply .github/release-drafter.yml:5]: Yes. Such configuration basically uses the organization-wide https://github.com/jenkinsci/.github/blob/master/.github/release-drafter.yml configuration

**Pull request merged**